golang/pkg/builder/container/container_builder.go

```diff
@@ -576,3 +596,21 @@ func (dc *DockerContainerBuilder) logWrite(message string) {
 		}
 	}
 }
+
+func CreateAndStart(dockerBuilder *DockerContainerBuilder) error {
```

Comment:
Function like this one and the following would be better off on the builder struct itself, imo.
Why have you chosen this solution instead?

Reply:
I looks like a helper function this way, that's why I did not put it in the Builder. But I can see why you'd move it inside. I'll fix that.